index.html

```diff
@@ -0,0 +1,68 @@
+<html>
+    <head>
+        <title>Youtube</title>
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+    <body>
+        <nav>
+            <div>
+                <image src="https://upload.wikimedia.org/wikipedia/commons/e/e1/Logo_of_YouTube_%282015-2017%29.svg" height= "100px" width="100px">
+            </div>
+
+            <div>
+                <input type="text" name="search" placeholder="search">
+            </div>
+            <div>
+                <button type="button" ><i class="fa fa-search"></i></button>
+            </div>
+
+            <div id="add-video-playlist">
+                <i class="fa fa-file-video-o" aria-hidden="true" style="font-size:24px"></i>
+            </div>
+
+            <div id="bell">
+                <i class="fa fa-bell" style="font-size:24px"></i>
+            </div>
+
+            <div id="profile-pic">
+                <img src="https://thumbor.forbes.com/thumbor/fit-in/416x416/filters%3Aformat%28jpg%29/https%3A%2F%2Fspecials-images.forbesimg.com%2Fimageserve%2F593b2e4b31358e03e55a0e8c%2F0x0.jpg%3Fbackground%3D000000%26cropX1%3D634%26cropX2%3D2468%26cropY1%3D39%26cropY2%3D1874">
+            </div>
+
+            
+        </nav>
+        <section id="review-info">
+            <div>
+            <div>
+                <iframe width="560" height="315" 
+                src="https://www.youtube.com/embed/Vx7YkKpt-J4" 
+                frameborder="0" allow="accelerometer; 
+                autoplay; clipboard-write; encrypted-media; 
+                gyroscope; picture-in-picture" allowfullscreen></iframe> 
+            </div>
+            <div><p>15,854 views</p></div>
+            <div><p><i class="fa fa-user" aria-hidden="true" style="font-size:24px"></i></p></div>
+            <div id="profile-record">
+                <div id="video-title"><p>Video Title</p></div>
+                <div id="like"><p><i class="fa fa-thumbs-up" aria-hidden="true"></i></p></div>
+                <div id="dislike"><p><i class="fa fa-thumbs-down" aria-hidden="true"></i></p></div>
+                <div id="share"><button><i class="fa fa-share" aria-hidden="true"></i> Share</button></div>
+                <div id="subscribe"><button>Subscribe</button></div>
+            </div>
+            </div>
+            <aside>
+                <p>
+                    <a target="_blank" href="https://www.youtube.com/watch?v=zlJDTxahav0">
+                        <img src="https://e00-marca.uecdn.es/assets/multimedia/imagenes/2020/05/22/15901557765954.jpg" height="100px" wight="100px" alt="Forest">
+                    </a>
+                </p>
+                <div display="down">
+                    <p> Selena Gomez - Lose You To Love Me (Official Music Video)</p>
+                    <p>304,229,255 views</p>
+                </div>
+            </aside>
```

Comment:
- [ ] **OPTIONAL** Your pages looks a little blank with only one recommended video, you can copy paste this recommended video section to fill with more content